doc/installation.rst

```diff
@@ -18,56 +18,62 @@ Linux
 ~~~~~
 
 We provide pre-built binaries for some platforms from the Git ``master``
-branch. Artifacts are distributed as TAR archives which can be unpacked to any
-location::
+branch. Artifacts are distributed as DEB or RPM distribution packages, or TAR archives; TAR archives need to be unpacked to `/opt/spicy`::
 
-    # Unpack archive to e.g., /opt/spicy overwriting any previous installation.
-    # Writing to /opt/spicy likely requires superuser priviledges.
-    # The path /opt/spicy can be replaced with a custom path.
+    # Using DEB distribution packages.
+    dpkg --install spicy.deb
```

Comment:
I don't recall what we said about the names: can we get version numbers into them? So for releases it would be `spicy-1.0.0.{tar.gz,deb,rpm}`, for master/nightly a date would be nice.

Reply:
We want to link to packages in the docs, and I am unsure how to do that without too much complications if we include version numbers. If we'd only link to artifacts for releases we could use the existing release automation for that, but for builds from master it seems we'd need to some form of templating in the documentation. Do you know if there's something in Sphinx we could use?

Reply:
Then lets do it just for releases. Leaving master builds at some generic name (say `spicy-dev.tbz`) is fine, less important for that.